promise/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "js-interview-promise",
+  "version": "1.0.0",
+  "scripts": {
+    "promise": "tsx --watch promiseTimeout.ts"
+  },
+  "author": "",
+  "license": "ISC",
```

Comment:
It's nice you have a license for an empty IIFE 👍🏽 
Maybe you would like to add a Readme as well

Reply:
This part is more verbal thing.